.pre-commit-config.yaml

```
@@ -83,7 +83,7 @@ repos:
         name: shellcheck
         language: docker_image
         entry: koalaman/shellcheck:v0.7.1
-        files: ".sh"
+        files: ".sh$"
```

```suggestion
        files: "\.sh$"
```

This would probably be even more accurate?

---

```suggestion
        files: "[.]sh$"
```

or rather this, cause I'm not sure how to escape